### PR TITLE
Feature/usability update 20190513

### PR DESCRIPTION
**Pull request by hendersonanalytics**
## Why?
- https://trello.com/c/6kXIEEqQ/1429-code-gov-site-map-generator-documentation-and-usability-update

## What Changed
- generate new sitemap
- update readme with more detailed directions
- add `dotenv` package so that `CODE_GOV_API_KEY` (and any other process.env variables that become necessary) can be declared in a .env file at the project's root ([dotenv package details](https://www.npmjs.com/package/dotenv))
- `console.log` out the number of projects retrieved from the api call during sitemap.xml file generation.

### CODE REVIEW

**Comment by saracope** [comment]
When I pulled down this branch the example.env did not come with it. Possibly because .env's are being ignored. Maybe switch it to env.example?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+## Quick Setup Instructions
+To run this app locally you'll need a code.gov API key.  If you don't have one, go to the following url to obtain one: https://developers.code.gov/key.html
+
+Once you have an API key, clone the repo and run the `npm install` command in the project's root directory to install all of the project’s dependencies.
+
+Run this terminal command: `cp .example.env .env`
+
+Replace `[your api key goes here]` in the `.env` file with your API key
+
+Run the `node index.js` terminal command to run the sitemap generation script.
```
Comment (saracope) [inline README.md]: Should just use the generate script: `npm run generate`.

Comment (hendersonanalytics) [reply README.md]: done

**Comment by hendersonanalytics** [comment]
@saracope I made the changes you requested. I deleted my local repo and cloned the remote down again and the example.env file was present this time.

**Comment by hendersonanalytics** [comment]
I think the original issue was with the terminal command, not the file name. It should have been `cp example.env .env` instead of `cp .example.env .env`

**Pull request merged**